tests/unit_tests/t_opt_parser.rb

```diff
@@ -4,13 +4,13 @@
 
 # Test the option parser
 class GitbotOptionTest < Minitest::Test
-  def set_option(hash, s)
```

Comment:
WARNING: I am not sure at all about this change, so waiting for @MalloZup input on how to fix this text correctly (as I understand it now checks the return code, but NOT the text).

Reply:
yop correct